prime-router/src/main/kotlin/fhirengine/translation/hl7/ConfigSchemaProcessor.kt

```diff
@@ -0,0 +1,136 @@
+package gov.cdc.prime.router.fhirengine.translation.hl7
+
+import gov.cdc.prime.router.fhirengine.translation.hl7.schema.ConfigSchemaElement
+import gov.cdc.prime.router.fhirengine.translation.hl7.utils.CustomContext
+import gov.cdc.prime.router.fhirengine.translation.hl7.utils.FhirPathUtils
+import org.apache.logging.log4j.kotlin.Logging
+import org.hl7.fhir.r4.model.Base
+import org.hl7.fhir.r4.model.Bundle
+
+abstract class ConfigSchemaProcessor : Logging {
+
+    /**
+     * Get the first valid string from the list of values specified in the schema for a given [element] using
+     * [bundle] and [context] starting at the [focusResource].
+     * @return the value for the element or an empty string if no value found
+     */
+    internal fun getValueAsString(
+        element: ConfigSchemaElement,
+        bundle: Bundle,
+        focusResource: Base,
+        context: CustomContext
+    ): String {
+        var retVal = ""
+        run findValue@{
```

Comment:
This was copied from the previously existing implementation of `getValue`, so I don't know for sure, but I think this just allows for us to resolve this block early if we find a valid value when going through the `element.value.forEach` iterations. A `break` there would probably achieve the same thing, but I think it's a little more readable by having a very explicit `return@findValue` instead of the somewhat ambiguous `break`.

Reply:
That's a cool Kotlin feature I didn't know about, thanks for the explanation!